### PR TITLE
pcb2gcode: vendor boost 1.66.0

### PR DESCRIPTION
**Pull request by lembacon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by lembacon** [comment]
cc @DomT4 @fxcoudert

**Comment by lembacon** [comment]
The `Reviewers` on this page keeps loading…

File: Formula/pcb2gcode.rb [hunk 1]
```diff
@@ -21,15 +21,59 @@ class Pcb2gcode < Formula
...
   depends_on "gtkmm"
   depends_on "gerbv"
 
+  resource "boost" do
+    url "https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.bz2"
+    sha256 "5721818253e6a0989583192f96782c4a98eb6204965316df9f5ad75819225ca9"
+  end
+
   def install
+    resource("boost").stage do
```
Comment (fxcoudert) [inline Formula/pcb2gcode.rb]: Add a comment linking to the bug report, and explaining that this can be replaced when the bug is fixed.

**Review by DomT4 — APPROVED** [review]
I'm happy once FX's comment is addressed & CI is happy.

File: Formula/pcb2gcode.rb (hunk 1 above)
Comment (lembacon) [reply Formula/pcb2gcode.rb]: Done. https://github.com/Homebrew/homebrew-core/pull/30926/files#diff-104bdf5db5756d516c4cc0e80e55e48aR27

**Pull request closed without merging**